tools/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualAppInfo.scala

```diff
@@ -136,6 +136,20 @@ class QualAppInfo(
     probNotDataset.values.flatten.toSet.mkString(":")
   }
 
+  private def getAllPotentialProblems(getPotentialProb: String, nestedComplex: String): String = {
```

Comment:
rename getPotentialProb to something useful, that is just a string to its not getting anything